core/src/main/java/org/apache/carbondata/core/datamap/DataMapUtil.java

```diff
@@ -159,4 +160,13 @@ private static DistributableDataMapFormat createDataMapJob(CarbonTable carbonTab
     return ssm.getValidAndInvalidSegments();
   }
 
+  /**
+   * Check if carbonTable is mv datamap table
+   */
+  public static Boolean isMVdatamapTable(CarbonTable carbonTable) {
```

Comment:
Change this method name isMVDatamapTable

Reply:
done